code/game/objects/items/weapons/cigs.dm

```diff
@@ -374,7 +377,7 @@ LIGHTERS ARE IN LIGHTERS.DM
 /obj/item/clothing/mask/cigarette/cigar/attackby(obj/item/I, mob/user, params)
 	if(istype(I, /obj/item/reagent_containers))
 		return
-	if(istype(I, /obj/item/match) || istype(I, /obj/item/lighter/zippo))
+	if(istype(I, /obj/item/match) || istype(I, /obj/item/lighter/zippo || istype(I, /obj/item/nullrod/godhand)))
```

Comment:
Not saying this is bad, but would lighting a pipe with your finger really be civilized enough for a pipe?

Reply:
Civilized, perhaps no, but it's niche and amusing enough. Besides, whole reason matches only work with pipes is because you can't exactly light a pipe with a lighter.

Reply:
Good enough for me

> Besides, whole reason matches only work with pipes is because you can't exactly light a pipe with a lighter.

And you can with a zippo? It's about style and sending a message.